src/Events/HealthyBackupWasFound.php

```diff
@@ -6,7 +6,7 @@
 
 class HealthyBackupWasFound
 {
-    /** @var \Spatie\Backup\BackupDestination\BackupDestinationsStatus */
```

Comment:
Could you remove the FQCN changes? We prefer to have namespaces in our docblocks.